docs/developer/discounts/promotions.mdx

```diff
@@ -55,7 +55,7 @@ price will not be instantly visible after promotion creation.
 The discounts from order rules are automatically applied during checkout or draft orders when certain conditions are met.
 Only the discount from the promotion rule that provides the maximum savings for the customer is applied.
 
-The order discounts can be combined with catalogue discounts, and in such cases, the discount is calculated based on the discounted price.
+The order discounts can be combined with [`line-level discounts`](developer/discounts/overview.mdx#level), and in such cases, the discount is calculated based on the discounted price.
```

Comment:
Are you sure that order discount is calculated after applying `specific product voucher`?

Reply:
You are right. The order promotion doesn't combine with vouchers. I am closing the ticket then